recipe/meta.yaml

```diff
@@ -211,3 +173,4 @@ extra:
     - leahecole
     - parthea
     - tswast
+    - xylar
```

Comment:
I'm hoping you will be okay with me joining as a maintainer.

Reply:
Thanks for volunteering!